---
annotation-target: JBM4-2-328.pdf
---


>%%
>```annotation-json
>{"created":"2022-09-07T01:45:00.553Z","updated":"2022-09-07T01:45:00.553Z","document":{"title":"Comparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative ProteomicsComparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative Proteomics","link":[{"href":"urn:x-pdf:403d35e0fa82a955503135f820802624"},{"href":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf"}],"documentFingerprint":"403d35e0fa82a955503135f820802624"},"uri":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","selector":[{"type":"TextPositionSelector","start":32,"end":86},{"type":"TextQuoteSelector","exact":"Osteoclasts DerivedFrom Murine Bone Marrow Macrophages","prefix":"Comparative Characterization of ","suffix":" and RAW 264.7Cells Using Quanti"}]}]}
>```
>%%
>*%%PREFIX%%Comparative Characterization of%%HIGHLIGHT%% ==Osteoclasts DerivedFrom Murine Bone Marrow Macrophages== %%POSTFIX%%and RAW 264.7Cells Using Quanti*
>%%LINK%%[[#^9egici9f0j|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^9egici9f0j


>%%
>```annotation-json
>{"created":"2022-09-07T01:45:06.461Z","updated":"2022-09-07T01:45:06.461Z","document":{"title":"Comparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative ProteomicsComparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative Proteomics","link":[{"href":"urn:x-pdf:403d35e0fa82a955503135f820802624"},{"href":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf"}],"documentFingerprint":"403d35e0fa82a955503135f820802624"},"uri":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","selector":[{"type":"TextPositionSelector","start":112,"end":135},{"type":"TextQuoteSelector","exact":"Quantitative Proteomics","prefix":"phages and RAW 264.7Cells Using ","suffix":"Andrew YH Ng,1,2,3 Chengjian Tu,"}]}]}
>```
>%%
>*%%PREFIX%%phages and RAW 264.7Cells Using%%HIGHLIGHT%% ==Quantitative Proteomics== %%POSTFIX%%Andrew YH Ng,1,2,3 Chengjian Tu,*
>%%LINK%%[[#^inckfbs61yn|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^inckfbs61yn


>%%
>```annotation-json
>{"created":"2022-09-07T01:48:17.217Z","updated":"2022-09-07T01:48:17.217Z","document":{"title":"Comparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative ProteomicsComparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative Proteomics","link":[{"href":"urn:x-pdf:403d35e0fa82a955503135f820802624"},{"href":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf"}],"documentFingerprint":"403d35e0fa82a955503135f820802624"},"uri":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","selector":[{"type":"TextPositionSelector","start":5066,"end":5573},{"type":"TextQuoteSelector","exact":"While BMMs are primarycells most frequently isolated from the marrow cavity of rodentlong bones,(5) RAW264.7 cells are a transformed macrophage-like cell line derived from the lymphoma of a male BALB/c mouseinfected by the Abelson murine leukemia virus (A-muLV).(6,7) Theretrovirus encodes an oncogenic form of the Abelson kinase, v-Abl, which is a fusion protein where portions of the retroviralGag protein substitute regions of the SH3 domain of c-Abl,rendering the tyrosine kinase constitutively active.(","prefix":"on cell signaling and behavior. ","suffix":"8–12) Studieshave shown that the"}]}]}
>```
>%%
>*%%PREFIX%%on cell signaling and behavior.%%HIGHLIGHT%% ==While BMMs are primarycells most frequently isolated from the marrow cavity of rodentlong bones,(5) RAW264.7 cells are a transformed macrophage-like cell line derived from the lymphoma of a male BALB/c mouseinfected by the Abelson murine leukemia virus (A-muLV).(6,7) Theretrovirus encodes an oncogenic form of the Abelson kinase, v-Abl, which is a fusion protein where portions of the retroviralGag protein substitute regions of the SH3 domain of c-Abl,rendering the tyrosine kinase constitutively active.(== %%POSTFIX%%8–12) Studieshave shown that the*
>%%LINK%%[[#^ri76eno2ecj|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^ri76eno2ecj


>%%
>```annotation-json
>{"created":"2022-09-07T01:49:20.798Z","updated":"2022-09-07T01:49:20.798Z","document":{"title":"Comparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative ProteomicsComparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative Proteomics","link":[{"href":"urn:x-pdf:403d35e0fa82a955503135f820802624"},{"href":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf"}],"documentFingerprint":"403d35e0fa82a955503135f820802624"},"uri":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","selector":[{"type":"TextPositionSelector","start":5605,"end":5952},{"type":"TextQuoteSelector","exact":" expression of v-Abl leads to the constitutiveactivation of mitogenic signaling pathways (e.g. Ras, Jak-Stat,JNK, and PI3K), resulting in uncontrolled cell proliferation—a keyfeature of cell transformation.(11) In human patients, the fusionprotein composed of the breakpoint cluster region (BCR) andABL1 (BCR-ABL) causes chronic myeloid leukemia.(","prefix":"8–12) Studieshave shown that the","suffix":"13) Patientsreceiving inhibitors"}]}]}
>```
>%%
>*%%PREFIX%%8–12) Studieshave shown that the%%HIGHLIGHT%% ==expression of v-Abl leads to the constitutiveactivation of mitogenic signaling pathways (e.g. Ras, Jak-Stat,JNK, and PI3K), resulting in uncontrolled cell proliferation—a keyfeature of cell transformation.(11) In human patients, the fusionprotein composed of the breakpoint cluster region (BCR) andABL1 (BCR-ABL) causes chronic myeloid leukemia.(== %%POSTFIX%%13) Patientsreceiving inhibitors*
>%%LINK%%[[#^mfp2svpvh38|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^mfp2svpvh38


>%%
>```annotation-json
>{"created":"2022-09-07T01:50:12.146Z","updated":"2022-09-07T01:50:12.146Z","document":{"title":"Comparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative ProteomicsComparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative Proteomics","link":[{"href":"urn:x-pdf:403d35e0fa82a955503135f820802624"},{"href":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf"}],"documentFingerprint":"403d35e0fa82a955503135f820802624"},"uri":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","selector":[{"type":"TextPositionSelector","start":7907,"end":8721},{"type":"TextQuoteSelector","exact":"Proteomics is a powerful tool that has led to numerousdiscoveries of proteins and biological processes that drive OCdifferentiation.(26) Notably, this technique was recently used tomap the podosome proteome and helped to advance ourunderstanding of determinants in the macrophage multi-nucleation process.(27,28) In other examples, proteomics analysisidentified changes in the expression of proteins involved inmetabolism and redirection of energy flow toward boneresorption in OCs.(29) Proteomics can therefore provide a broadyet informative overview of the systemic changes in thedifferentiating OC. We employed a robust and high-throughputliquid-chromatography mass spectrometry (LC-MS)-based pro-teomics approach to characterize the global protein changes ofOCs at different stages of the reprogramming process","prefix":"d its related signalingpathways.","suffix":". Ourfindings provide the first "}]}]}
>```
>%%
>*%%PREFIX%%d its related signalingpathways.%%HIGHLIGHT%% ==Proteomics is a powerful tool that has led to numerousdiscoveries of proteins and biological processes that drive OCdifferentiation.(26) Notably, this technique was recently used tomap the podosome proteome and helped to advance ourunderstanding of determinants in the macrophage multi-nucleation process.(27,28) In other examples, proteomics analysisidentified changes in the expression of proteins involved inmetabolism and redirection of energy flow toward boneresorption in OCs.(29) Proteomics can therefore provide a broadyet informative overview of the systemic changes in thedifferentiating OC. We employed a robust and high-throughputliquid-chromatography mass spectrometry (LC-MS)-based pro-teomics approach to characterize the global protein changes ofOCs at different stages of the reprogramming process== %%POSTFIX%%. Ourfindings provide the first*
>%%LINK%%[[#^sncuvrrq79|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^sncuvrrq79


>%%
>```annotation-json
>{"created":"2022-09-07T01:54:13.117Z","updated":"2022-09-07T01:54:13.117Z","document":{"title":"Comparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative ProteomicsComparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative Proteomics","link":[{"href":"urn:x-pdf:403d35e0fa82a955503135f820802624"},{"href":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf"}],"documentFingerprint":"403d35e0fa82a955503135f820802624"},"uri":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","selector":[{"type":"TextPositionSelector","start":11688,"end":11827},{"type":"TextQuoteSelector","exact":"The IonStar LC-MS experimental pipeline was developed andoptimized in a previous study.(35) Details can be found in theSupplemental Methods","prefix":"tandem mass spectrometryanalysis","suffix":". A stringent set of criteria in"}]}]}
>```
>%%
>*%%PREFIX%%tandem mass spectrometryanalysis%%HIGHLIGHT%% ==The IonStar LC-MS experimental pipeline was developed andoptimized in a previous study.(35) Details can be found in theSupplemental Methods== %%POSTFIX%%. A stringent set of criteria in*
>%%LINK%%[[#^j4y88ba6wko|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^j4y88ba6wko


>%%
>```annotation-json
>{"created":"2022-09-07T01:54:25.901Z","updated":"2022-09-07T01:54:25.901Z","document":{"title":"Comparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative ProteomicsComparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative Proteomics","link":[{"href":"urn:x-pdf:403d35e0fa82a955503135f820802624"},{"href":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf"}],"documentFingerprint":"403d35e0fa82a955503135f820802624"},"uri":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","selector":[{"type":"TextPositionSelector","start":12090,"end":12285},{"type":"TextQuoteSelector","exact":" The mass spectrometryproteomics data have been deposited to the ProteomeXchangeConsortium via the Proteomics Identifications (PRIDE)(36) partnerrepository with the data set identifier PXD009610.","prefix":"e) was described previously.(35)","suffix":"Bioinformatics analysisIngenuity"}]}]}
>```
>%%
>*%%PREFIX%%e) was described previously.(35)%%HIGHLIGHT%% ==The mass spectrometryproteomics data have been deposited to the ProteomeXchangeConsortium via the Proteomics Identifications (PRIDE)(36) partnerrepository with the data set identifier PXD009610.== %%POSTFIX%%Bioinformatics analysisIngenuity*
>%%LINK%%[[#^ryjisn986wg|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^ryjisn986wg


>%%
>```annotation-json
>{"created":"2022-09-07T01:54:38.010Z","updated":"2022-09-07T01:54:38.010Z","document":{"title":"Comparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative ProteomicsComparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative Proteomics","link":[{"href":"urn:x-pdf:403d35e0fa82a955503135f820802624"},{"href":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf"}],"documentFingerprint":"403d35e0fa82a955503135f820802624"},"uri":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","selector":[{"type":"TextPositionSelector","start":12285,"end":12307},{"type":"TextQuoteSelector","exact":"Bioinformatics analysi","prefix":"e data set identifier PXD009610.","suffix":"sIngenuity Pathway Analysis (Qia"}]}]}
>```
>%%
>*%%PREFIX%%e data set identifier PXD009610.%%HIGHLIGHT%% ==Bioinformatics analysi== %%POSTFIX%%sIngenuity Pathway Analysis (Qia*
>%%LINK%%[[#^lld79mag1x|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^lld79mag1x


>%%
>```annotation-json
>{"created":"2022-09-07T01:55:06.106Z","updated":"2022-09-07T01:55:06.106Z","document":{"title":"Comparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative ProteomicsComparative Characterization of Osteoclasts Derived From Murine Bone Marrow Macrophages and RAW 264.7 Cells Using Quantitative Proteomics","link":[{"href":"urn:x-pdf:403d35e0fa82a955503135f820802624"},{"href":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf"}],"documentFingerprint":"403d35e0fa82a955503135f820802624"},"uri":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220830andrewng/JBM4-2-328.pdf","selector":[{"type":"TextPositionSelector","start":12308,"end":12727},{"type":"TextQuoteSelector","exact":"Ingenuity Pathway Analysis (Qiagen, Redwood City, CA, USA) andInnateDB(37) were used to perform gene ontology (GO) termanalysis. Hierarchical clustering analysis and heat map visualizationwere performed using the heatplot function in R Package made4.Fuzzy C-means clustering was performed using the cmeansfunction in R Package e1071, and the results were visualized using the R package ggplot2 and viridis color palette","prefix":"XD009610.Bioinformatics analysis","suffix":".Western blotCells were cultured"}]}]}
>```
>%%
>*%%PREFIX%%XD009610.Bioinformatics analysis%%HIGHLIGHT%% ==Ingenuity Pathway Analysis (Qiagen, Redwood City, CA, USA) andInnateDB(37) were used to perform gene ontology (GO) termanalysis. Hierarchical clustering analysis and heat map visualizationwere performed using the heatplot function in R Package made4.Fuzzy C-means clustering was performed using the cmeansfunction in R Package e1071, and the results were visualized using the R package ggplot2 and viridis color palette== %%POSTFIX%%.Western blotCells were cultured*
>%%LINK%%[[#^pdrwoamnrdh|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^pdrwoamnrdh
