---
annotation-target: blood.2022017361.pdf
---


>%%
>```annotation-json
>{"created":"2022-09-15T01:56:25.217Z","updated":"2022-09-15T01:56:25.217Z","document":{"title":"blood.2022017361.pdf","link":[{"href":"urn:x-pdf:d4d389180d51761c91bd8f18e6b8ea88"},{"href":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf"}],"documentFingerprint":"d4d389180d51761c91bd8f18e6b8ea88"},"uri":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","selector":[{"type":"TextPositionSelector","start":140,"end":160},{"type":"TextQuoteSelector","exact":"Single Cell Genomics","prefix":"776-0545editorial@hematology.org","suffix":" - a new chapter in how technolo"}]}]}
>```
>%%
>*%%PREFIX%%776-0545editorial@hematology.org%%HIGHLIGHT%% ==Single Cell Genomics== %%POSTFIX%%- a new chapter in how technolo*
>%%LINK%%[[#^nmbzaye77aa|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^nmbzaye77aa


>%%
>```annotation-json
>{"created":"2022-09-15T01:57:50.451Z","updated":"2022-09-15T01:57:50.451Z","document":{"title":"blood.2022017361.pdf","link":[{"href":"urn:x-pdf:d4d389180d51761c91bd8f18e6b8ea88"},{"href":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf"}],"documentFingerprint":"d4d389180d51761c91bd8f18e6b8ea88"},"uri":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","selector":[{"type":"TextPositionSelector","start":2476,"end":2995},{"type":"TextQuoteSelector","exact":"This early adoption of technology included methods for the examination of phenotype and function of single cells, alongside detection of chromosomal rearrangements in single malignant haematopoietic cells.  FACS (Fluorescence ‐activated cell sorting) using monoclonal antibodies 12 allowed isolation and characterisation of cells defined by specific combinations of surface markers, while in vitro colony assays3,4 and single cell transplantation permitted functional assessment of lineage and proliferative potential 5","prefix":"ny other clinical specialities. ","suffix":"–7. The advent of single cell re"}]}]}
>```
>%%
>*%%PREFIX%%ny other clinical specialities.%%HIGHLIGHT%% ==This early adoption of technology included methods for the examination of phenotype and function of single cells, alongside detection of chromosomal rearrangements in single malignant haematopoietic cells.  FACS (Fluorescence ‐activated cell sorting) using monoclonal antibodies 12 allowed isolation and characterisation of cells defined by specific combinations of surface markers, while in vitro colony assays3,4 and single cell transplantation permitted functional assessment of lineage and proliferative potential 5== %%POSTFIX%%–7. The advent of single cell re*
>%%LINK%%[[#^ecrg94cgy4|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^ecrg94cgy4


>%%
>```annotation-json
>{"created":"2022-09-15T01:59:16.407Z","updated":"2022-09-15T01:59:16.407Z","document":{"title":"blood.2022017361.pdf","link":[{"href":"urn:x-pdf:d4d389180d51761c91bd8f18e6b8ea88"},{"href":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf"}],"documentFingerprint":"d4d389180d51761c91bd8f18e6b8ea88"},"uri":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","selector":[{"type":"TextPositionSelector","start":3341,"end":3699},{"type":"TextQuoteSelector","exact":"The study of normal haematopoiesis has been transformed in recent years by the application of single cell genomics technologies which permit unbiased interrogation of molecular profiles at unprecedented resolution. Using massively parallel scRNA‐seq (single cell RNA sequencing) combined with indexed FACS sorting, chromatin profiling and functional assays, ","prefix":"le cell genomics techniques.    ","suffix":"Paul et al9 were able to charact"}]}]}
>```
>%%
>*%%PREFIX%%le cell genomics techniques.%%HIGHLIGHT%% ==The study of normal haematopoiesis has been transformed in recent years by the application of single cell genomics technologies which permit unbiased interrogation of molecular profiles at unprecedented resolution. Using massively parallel scRNA‐seq (single cell RNA sequencing) combined with indexed FACS sorting, chromatin profiling and functional assays,== %%POSTFIX%%Paul et al9 were able to charact*
>%%LINK%%[[#^d2gjluf2og|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^d2gjluf2og


>%%
>```annotation-json
>{"created":"2022-09-15T02:00:52.694Z","updated":"2022-09-15T02:00:52.694Z","document":{"title":"blood.2022017361.pdf","link":[{"href":"urn:x-pdf:d4d389180d51761c91bd8f18e6b8ea88"},{"href":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf"}],"documentFingerprint":"d4d389180d51761c91bd8f18e6b8ea88"},"uri":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","selector":[{"type":"TextPositionSelector","start":8977,"end":9578},{"type":"TextQuoteSelector","exact":"How far are we from translating these techniques into clinical practice, and where could the most value be added?  In the short‐to medium term, as throughput approximates that of high‐resolution flow cytometry, likely the greatest added value for single‐cell genomics may be gained from clonal tracking and assessment of remission or residual disease, complementing current flow cytometry and bulk sequencing minimal residual disease (MRD) methods by detecting rare populations, demonstrating subclonal hierarchies and non‐genomic heterogeneity, predicting prognosis or response to further treatment. ","prefix":"ing the MPN stem cell.          ","suffix":" While Downloaded from http://as"}]}]}
>```
>%%
>*%%PREFIX%%ing the MPN stem cell.%%HIGHLIGHT%% ==How far are we from translating these techniques into clinical practice, and where could the most value be added?  In the short‐to medium term, as throughput approximates that of high‐resolution flow cytometry, likely the greatest added value for single‐cell genomics may be gained from clonal tracking and assessment of remission or residual disease, complementing current flow cytometry and bulk sequencing minimal residual disease (MRD) methods by detecting rare populations, demonstrating subclonal hierarchies and non‐genomic heterogeneity, predicting prognosis or response to further treatment.== %%POSTFIX%%While Downloaded from http://as*
>%%LINK%%[[#^09odsi43bj1m|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^09odsi43bj1m


>%%
>```annotation-json
>{"created":"2022-09-15T02:01:56.307Z","updated":"2022-09-15T02:01:56.307Z","document":{"title":"blood.2022017361.pdf","link":[{"href":"urn:x-pdf:d4d389180d51761c91bd8f18e6b8ea88"},{"href":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf"}],"documentFingerprint":"d4d389180d51761c91bd8f18e6b8ea88"},"uri":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220915/blood.2022017361.pdf","selector":[{"type":"TextPositionSelector","start":10319,"end":10734},{"type":"TextQuoteSelector","exact":" are of tumour heterogeneity and microenvironmental factors at diagnosis which may predict treatment response and tumour evolution.  In the medium to long term, further single cell multiomic and spatial studies characterising mechanisms of disease initiation and progression are likely to elucidate treatment vulnerabilities and novel risk stratifications which will inform treatment choice at diagnosis and relapse","prefix":"ur reviews in the current series","suffix":".  At present single cell genomi"}]}]}
>```
>%%
>*%%PREFIX%%ur reviews in the current series%%HIGHLIGHT%% ==are of tumour heterogeneity and microenvironmental factors at diagnosis which may predict treatment response and tumour evolution.  In the medium to long term, further single cell multiomic and spatial studies characterising mechanisms of disease initiation and progression are likely to elucidate treatment vulnerabilities and novel risk stratifications which will inform treatment choice at diagnosis and relapse== %%POSTFIX%%.  At present single cell genomi*
>%%LINK%%[[#^rlsgkkuq0n|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^rlsgkkuq0n
