---
annotation-target: 41375_2021_Article_1316.pdf
---


>%%
>```annotation-json
>{"created":"2022-09-07T13:01:56.132Z","updated":"2022-09-07T13:01:56.132Z","document":{"title":"Investigation of measurable residual disease in acute myeloid leukemia by DNA methylation patterns","link":[{"href":"urn:x-pdf:e885eaa8f7caad39c45cd75451dadcf0"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf"}],"documentFingerprint":"e885eaa8f7caad39c45cd75451dadcf0"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","selector":[{"type":"TextPositionSelector","start":488,"end":743},{"type":"TextQuoteSelector","exact":"addressed by highly sensitive PCR- or sequencing-based screening of specific mutations, or by multiparametric flowcytometry. However, not all patients have suitable mutations and heterogeneity of surface markers hampers standardization inclinical routine.","prefix":"mains challenging. It isusually ","suffix":" In this study, we propose an al"}]}]}
>```
>%%
>*%%PREFIX%%mains challenging. It isusually%%HIGHLIGHT%% ==addressed by highly sensitive PCR- or sequencing-based screening of specific mutations, or by multiparametric flowcytometry. However, not all patients have suitable mutations and heterogeneity of surface markers hampers standardization inclinical routine.== %%POSTFIX%%In this study, we propose an al*
>%%LINK%%[[#^80bsqo4mlnn|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^80bsqo4mlnn


>%%
>```annotation-json
>{"created":"2022-09-07T13:02:48.105Z","updated":"2022-09-07T13:02:48.105Z","document":{"title":"Investigation of measurable residual disease in acute myeloid leukemia by DNA methylation patterns","link":[{"href":"urn:x-pdf:e885eaa8f7caad39c45cd75451dadcf0"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf"}],"documentFingerprint":"e885eaa8f7caad39c45cd75451dadcf0"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","selector":[{"type":"TextPositionSelector","start":2048,"end":2396},{"type":"TextQuoteSelector","exact":"Early and accurate detection ofremaining leukemia cells—previously referred to as “minimalresidual disease”(MRD), and now more appropriately termed“measurable residual disease”with the same acronym—has beenshown to be a strong prognostic indicator for relapse of AML, andmight therefore be used as parameter for deciding on post-remission treatment","prefix":"ntually experience relapse [1]. ","suffix":" [2, 3].Assessment of MRD is bas"}]}]}
>```
>%%
>*%%PREFIX%%ntually experience relapse [1].%%HIGHLIGHT%% ==Early and accurate detection ofremaining leukemia cells—previously referred to as “minimalresidual disease”(MRD), and now more appropriately termed“measurable residual disease”with the same acronym—has beenshown to be a strong prognostic indicator for relapse of AML, andmight therefore be used as parameter for deciding on post-remission treatment== %%POSTFIX%%[2, 3].Assessment of MRD is bas*
>%%LINK%%[[#^8f2p5da8nlt|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^8f2p5da8nlt


>%%
>```annotation-json
>{"created":"2022-09-07T13:03:26.549Z","updated":"2022-09-07T13:03:26.549Z","document":{"title":"Investigation of measurable residual disease in acute myeloid leukemia by DNA methylation patterns","link":[{"href":"urn:x-pdf:e885eaa8f7caad39c45cd75451dadcf0"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf"}],"documentFingerprint":"e885eaa8f7caad39c45cd75451dadcf0"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","selector":[{"type":"TextPositionSelector","start":2404,"end":3380},{"type":"TextQuoteSelector","exact":"Assessment of MRD is based on detection of specific immuno-phenotypic [4], or genotypic [5] aberrations. This is commonlyperformed by multiparameter flow cytometry (MFC) of leukemia-associated immunophenotypes, and detection of specific mutations by real-time quantitative PCR (RT-qPCR), digital droplet PCR, or nextgeneration sequencing (NGS) [1, 3]. So far, RT-qPCR is considered asa gold standard for detection of MRD, however, it is applicable toonly 40–60% of AML patients as not all patients have suitabletranslocations or mutations [3, 6]. Frameshift mutations ofnucleophosmin 1 (NPM1) are one of the most frequent molecularabnormalities in AML that remain relatively stable in the course ofthe disease and therefore represent an ideal target for RT-qPCR MRDmonitoring [7, 8]. Immunophenotypic detection of MRD canpotentially investigate more than 85% of AML cases, but thisapproach lacks standardization of selected markers, instrumentsettings or reporting thresholds ","prefix":"post-remission treatment [2, 3].","suffix":"[9].Aberrant DNA methylation (DN"}]}]}
>```
>%%
>*%%PREFIX%%post-remission treatment [2, 3].%%HIGHLIGHT%% ==Assessment of MRD is based on detection of specific immuno-phenotypic [4], or genotypic [5] aberrations. This is commonlyperformed by multiparameter flow cytometry (MFC) of leukemia-associated immunophenotypes, and detection of specific mutations by real-time quantitative PCR (RT-qPCR), digital droplet PCR, or nextgeneration sequencing (NGS) [1, 3]. So far, RT-qPCR is considered asa gold standard for detection of MRD, however, it is applicable toonly 40–60% of AML patients as not all patients have suitabletranslocations or mutations [3, 6]. Frameshift mutations ofnucleophosmin 1 (NPM1) are one of the most frequent molecularabnormalities in AML that remain relatively stable in the course ofthe disease and therefore represent an ideal target for RT-qPCR MRDmonitoring [7, 8]. Immunophenotypic detection of MRD canpotentially investigate more than 85% of AML cases, but thisapproach lacks standardization of selected markers, instrumentsettings or reporting thresholds== %%POSTFIX%%[9].Aberrant DNA methylation (DN*
>%%LINK%%[[#^byz9n3ge0fu|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^byz9n3ge0fu


>%%
>```annotation-json
>{"created":"2022-09-07T13:07:14.905Z","updated":"2022-09-07T13:07:14.905Z","document":{"title":"Investigation of measurable residual disease in acute myeloid leukemia by DNA methylation patterns","link":[{"href":"urn:x-pdf:e885eaa8f7caad39c45cd75451dadcf0"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf"}],"documentFingerprint":"e885eaa8f7caad39c45cd75451dadcf0"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","selector":[{"type":"TextPositionSelector","start":3608,"end":4001},{"type":"TextQuoteSelector","exact":"The epigenetic land-scape of AML is particularly modified by mutations andepimutations in epigenetic modifiers, such as DNA methyltrans-ferase DNMT3A [13–15] and TET2 [16]. Several studies demon-strated that DNAm can be utilized as a biomarker for AMLdiagnosis, classification, and prognostic stratification [17–19]. Forexample, AML-subtypes reveal specific genome-wide DNAmsignatures [20, 21]","prefix":"er-associated CpG-islands [12]. ","suffix":".In this study, we followed the "}]}]}
>```
>%%
>*%%PREFIX%%er-associated CpG-islands [12].%%HIGHLIGHT%% ==The epigenetic land-scape of AML is particularly modified by mutations andepimutations in epigenetic modifiers, such as DNA methyltrans-ferase DNMT3A [13–15] and TET2 [16]. Several studies demon-strated that DNAm can be utilized as a biomarker for AMLdiagnosis, classification, and prognostic stratification [17–19]. Forexample, AML-subtypes reveal specific genome-wide DNAmsignatures [20, 21]== %%POSTFIX%%.In this study, we followed the*
>%%LINK%%[[#^kinl2bsrvo|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^kinl2bsrvo


>%%
>```annotation-json
>{"created":"2022-09-07T13:07:43.419Z","updated":"2022-09-07T13:07:43.419Z","document":{"title":"Investigation of measurable residual disease in acute myeloid leukemia by DNA methylation patterns","link":[{"href":"urn:x-pdf:e885eaa8f7caad39c45cd75451dadcf0"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf"}],"documentFingerprint":"e885eaa8f7caad39c45cd75451dadcf0"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1316.pdf","selector":[{"type":"TextPositionSelector","start":5546,"end":5729},{"type":"TextQuoteSelector","exact":": GSE40279 [22], GSE50660 [23] and GSE77716 [24]; and from bonemarrow of AML patients: The Cancer Genome Atlas (TCGA) [25], GSE58477(cytogenetic normal AMLs) [26] and GSE62298 [27]. T","prefix":"sets of blood from healthydonors","suffix":"o identify reliable andreproduci"}]}]}
>```
>%%
>*%%PREFIX%%sets of blood from healthydonors%%HIGHLIGHT%% ==: GSE40279 [22], GSE50660 [23] and GSE77716 [24]; and from bonemarrow of AML patients: The Cancer Genome Atlas (TCGA) [25], GSE58477(cytogenetic normal AMLs) [26] and GSE62298 [27]. T== %%POSTFIX%%o identify reliable andreproduci*
>%%LINK%%[[#^jbvc18e2tka|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^jbvc18e2tka
