---
annotation-target: 41375_2021_Article_1408.pdf
---


>%%
>```annotation-json
>{"created":"2022-09-07T09:15:41.240Z","updated":"2022-09-07T09:15:41.240Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":1987,"end":2575},{"type":"TextQuoteSelector","exact":"One of the most commonly mutated genes in AML isNucleophosmin 1 (NPM1). It plays a critical role in disease initiationand is utilized for molecular risk stratification in the recent EuropeanLeukemia Net 2017 (ELN2017) recommendations [1]. Mutated NMP1can be found in a third of all adult AML cases and up to 50–60% inAML with a normal karyotype [6, 7] and is considered a distinctdisease entity in the current WHO classification [8]. So far, differentmorphological subtypes of AML according to the FAB classification[3] have been associated with different frequencies of NPM1mutations [9]","prefix":"8;21), inv(16), or t(16;16)[5]. ","suffix":". However, the interpretation of"}]}]}
>```
>%%
>*%%PREFIX%%8;21), inv(16), or t(16;16)[5].%%HIGHLIGHT%% ==One of the most commonly mutated genes in AML isNucleophosmin 1 (NPM1). It plays a critical role in disease initiationand is utilized for molecular risk stratification in the recent EuropeanLeukemia Net 2017 (ELN2017) recommendations [1]. Mutated NMP1can be found in a third of all adult AML cases and up to 50–60% inAML with a normal karyotype [6, 7] and is considered a distinctdisease entity in the current WHO classification [8]. So far, differentmorphological subtypes of AML according to the FAB classification[3] have been associated with different frequencies of NPM1mutations [9]== %%POSTFIX%%. However, the interpretation of*
>%%LINK%%[[#^9evf54hp605|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^9evf54hp605


>%%
>```annotation-json
>{"created":"2022-09-07T09:16:15.173Z","updated":"2022-09-07T09:16:15.173Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":3390,"end":3534},{"type":"TextQuoteSelector","exact":"predicts NPM1 mutation status from bone marrowcytomorphology and unveils distinct morphologic features for theprediction of NPM1 mutation status","prefix":"s. Further, our modelaccurately ","suffix":".METHODSData set and molecular a"}]}]}
>```
>%%
>*%%PREFIX%%s. Further, our modelaccurately%%HIGHLIGHT%% ==predicts NPM1 mutation status from bone marrowcytomorphology and unveils distinct morphologic features for theprediction of NPM1 mutation status== %%POSTFIX%%.METHODSData set and molecular a*
>%%LINK%%[[#^7ozlyd7utf|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^7ozlyd7utf


>%%
>```annotation-json
>{"created":"2022-09-07T09:16:40.002Z","updated":"2022-09-07T09:16:40.002Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":3573,"end":3914},{"type":"TextQuoteSelector","exact":"We identified 1251 patients who have been newly diagnosed and treatedwith AML in the previously reported multicentric trials (AML96 [19]),AML2003 [20], AMLCG1999 [21], AML60+[22], AMLCG2008 [23], andSORAML [24]) or from the multicentric patient registry of the GermanStudy Alliance Leukemia (SAL, NCT03188874) via retrospective chartreview. ","prefix":"SData set and molecular analysis","suffix":"Eligibility criteria for the AML"}]}]}
>```
>%%
>*%%PREFIX%%SData set and molecular analysis%%HIGHLIGHT%% ==We identified 1251 patients who have been newly diagnosed and treatedwith AML in the previously reported multicentric trials (AML96 [19]),AML2003 [20], AMLCG1999 [21], AML60+[22], AMLCG2008 [23], andSORAML [24]) or from the multicentric patient registry of the GermanStudy Alliance Leukemia (SAL, NCT03188874) via retrospective chartreview.== %%POSTFIX%%Eligibility criteria for the AML*
>%%LINK%%[[#^8ak2gyzmbst|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^8ak2gyzmbst


>%%
>```annotation-json
>{"created":"2022-09-07T09:17:32.785Z","updated":"2022-09-07T09:17:32.785Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":6408,"end":6640},{"type":"TextQuoteSelector","exact":"an online segmentation and labeling platform that we developedfor the purpose of this work. A human-in-the-loop cell segmentation approachwas performed by hematologists with a Faster Region-based ConvolutionalNeural Net [28] (FRCNN)","prefix":"ere digitalized and uploaded to ","suffix":". First, initial segmentation wa"}]}]}
>```
>%%
>*%%PREFIX%%ere digitalized and uploaded to%%HIGHLIGHT%% ==an online segmentation and labeling platform that we developedfor the purpose of this work. A human-in-the-loop cell segmentation approachwas performed by hematologists with a Faster Region-based ConvolutionalNeural Net [28] (FRCNN)== %%POSTFIX%%. First, initial segmentation wa*
>%%LINK%%[[#^etuwu2rfcef|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^etuwu2rfcef


>%%
>```annotation-json
>{"created":"2022-09-07T09:18:49.537Z","updated":"2022-09-07T09:18:49.537Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":8211,"end":8319},{"type":"TextQuoteSelector","exact":" Model architecture for the distinctionbetween AML and healthy control samples was based on the Xception CNN","prefix":" final classification decisions.","suffix":"[31] utilizing transfer learning"}]}]}
>```
>%%
>*%%PREFIX%%final classification decisions.%%HIGHLIGHT%% ==Model architecture for the distinctionbetween AML and healthy control samples was based on the Xception CNN== %%POSTFIX%%[31] utilizing transfer learning*
>%%LINK%%[[#^9050thoyf4b|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^9050thoyf4b


>%%
>```annotation-json
>{"created":"2022-09-07T09:19:09.855Z","updated":"2022-09-07T09:19:09.855Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":8398,"end":8428},{"type":"TextQuoteSelector","exact":"BMS images (2560 × 1920 pixels","prefix":"itecture was modified to receive","suffix":") as input at the top level. Ful"}]}]}
>```
>%%
>*%%PREFIX%%itecture was modified to receive%%HIGHLIGHT%% ==BMS images (2560 × 1920 pixels== %%POSTFIX%%) as input at the top level. Ful*
>%%LINK%%[[#^s3ssd25esrl|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^s3ssd25esrl


>%%
>```annotation-json
>{"created":"2022-09-07T09:19:22.240Z","updated":"2022-09-07T09:19:22.240Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":8495,"end":8659},{"type":"TextQuoteSelector","exact":"048-dimensional output vectors of the core Xceptionarchitecture were established via hyperparameter optimization. Hyperpara-meters differed between individual model","prefix":"connectedoutput layers for the 2","suffix":"s. Step 4: For NPM1 status predi"}]}]}
>```
>%%
>*%%PREFIX%%connectedoutput layers for the 2%%HIGHLIGHT%% ==048-dimensional output vectors of the core Xceptionarchitecture were established via hyperparameter optimization. Hyperpara-meters differed between individual model== %%POSTFIX%%s. Step 4: For NPM1 status predi*
>%%LINK%%[[#^3lkhbgehiiv|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^3lkhbgehiiv


>%%
>```annotation-json
>{"created":"2022-09-07T09:19:46.091Z","updated":"2022-09-07T09:19:46.091Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":8670,"end":9015},{"type":"TextQuoteSelector","exact":"For NPM1 status prediction,transfer learning with a ResNet50 [32] pretrained on ImageNet was utilized onBMS images. The ResNet50-architecture was modified to use larger input sizes(2000 × 1500 pixels) and the output layer was reshaped to a fully connectedlayer with two neurons to accommodate the binary classification problembefore retraining. ","prefix":"ween individual models. Step 4: ","suffix":"Hyperparameter optimization was "}]}]}
>```
>%%
>*%%PREFIX%%ween individual models. Step 4:%%HIGHLIGHT%% ==For NPM1 status prediction,transfer learning with a ResNet50 [32] pretrained on ImageNet was utilized onBMS images. The ResNet50-architecture was modified to use larger input sizes(2000 × 1500 pixels) and the output layer was reshaped to a fully connectedlayer with two neurons to accommodate the binary classification problembefore retraining.== %%POSTFIX%%Hyperparameter optimization was*
>%%LINK%%[[#^ppmqpc7g9v8|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^ppmqpc7g9v8


>%%
>```annotation-json
>{"created":"2022-09-07T09:19:59.690Z","updated":"2022-09-07T09:19:59.690Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":9238,"end":9437},{"type":"TextQuoteSelector","exact":" DL models were implemented in Python version 3.7.9 withKeras version 2.3.0, TensorFlow version 2.1.2, and PyTorch version 1.5.0.Computations were performed using a high-performance computing system.","prefix":"sions for NPM1status prediction.","suffix":"Model performance and statistica"}]}]}
>```
>%%
>*%%PREFIX%%sions for NPM1status prediction.%%HIGHLIGHT%% ==DL models were implemented in Python version 3.7.9 withKeras version 2.3.0, TensorFlow version 2.1.2, and PyTorch version 1.5.0.Computations were performed using a high-performance computing system.== %%POSTFIX%%Model performance and statistica*
>%%LINK%%[[#^e88px0ke8hf|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^e88px0ke8hf


>%%
>```annotation-json
>{"created":"2022-09-07T09:20:44.069Z","updated":"2022-09-07T09:20:44.069Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":11101,"end":11254},{"type":"TextQuoteSelector","exact":" To compare NPM1 VAF the Mann–Whitney U test was used.Computational and statistical analysis was performed in Python (version3.7.9) and R (version 4.0.3)","prefix":"was withheld frommodel training.","suffix":".RESULTSDL accurately distinguis"}]}]}
>```
>%%
>*%%PREFIX%%was withheld frommodel training.%%HIGHLIGHT%% ==To compare NPM1 VAF the Mann–Whitney U test was used.Computational and statistical analysis was performed in Python (version3.7.9) and R (version 4.0.3)== %%POSTFIX%%.RESULTSDL accurately distinguis*
>%%LINK%%[[#^v8495zm7bjf|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^v8495zm7bjf


>%%
>```annotation-json
>{"created":"2022-09-07T09:21:12.900Z","updated":"2022-09-07T09:21:12.900Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":18940,"end":18963},{"type":"TextQuoteSelector","exact":"Patient characteristics","prefix":"ns in NPM1-mutated AML.Table 1. ","suffix":".Parameter All AML samples NPM1-"}]}]}
>```
>%%
>*%%PREFIX%%ns in NPM1-mutated AML.Table 1.%%HIGHLIGHT%% ==Patient characteristics== %%POSTFIX%%.Parameter All AML samples NPM1-*
>%%LINK%%[[#^xilpsjowyx|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^xilpsjowyx


>%%
>```annotation-json
>{"created":"2022-09-07T09:21:29.167Z","updated":"2022-09-07T09:21:29.167Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":14615,"end":14855},{"type":"TextQuoteSelector","exact":" The AML classification model achieved anaverage AUC for the precision–recall curve of 0.9691 (95% CI:0.9669–0.9713; Fig. 3A) and an average AUROC of 0.9699 (95% CI:0.9677–0.9721; Fig. 3B) with a corresponding micro-averageaccuracy of 0.91.","prefix":"ed hyperpara-meter optimization.","suffix":" Table 2A shows the distribution"}]}]}
>```
>%%
>*%%PREFIX%%ed hyperpara-meter optimization.%%HIGHLIGHT%% ==The AML classification model achieved anaverage AUC for the precision–recall curve of 0.9691 (95% CI:0.9669–0.9713; Fig. 3A) and an average AUROC of 0.9699 (95% CI:0.9677–0.9721; Fig. 3B) with a corresponding micro-averageaccuracy of 0.91.== %%POSTFIX%%Table 2A shows the distribution*
>%%LINK%%[[#^tw0f0k0uf8|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^tw0f0k0uf8


>%%
>```annotation-json
>{"created":"2022-09-07T09:22:58.764Z","updated":"2022-09-07T09:22:58.764Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":27071,"end":27334},{"type":"TextQuoteSelector","exact":" Pictures were taken at 50-fold magnification using a NikonECLIPSE E600 microscope with a mounted Nikon DSFi2 camera. Images were processed using Nikon Imaging Software Elements D4. Scale bar=10 μm. (ii) Heatmap of occlusion sensitivity maps generated in Python. ","prefix":" the May–Grunwald–Giemsa method.","suffix":"The scale indicates the importan"}]}]}
>```
>%%
>*%%PREFIX%%the May–Grunwald–Giemsa method.%%HIGHLIGHT%% ==Pictures were taken at 50-fold magnification using a NikonECLIPSE E600 microscope with a mounted Nikon DSFi2 camera. Images were processed using Nikon Imaging Software Elements D4. Scale bar=10 μm. (ii) Heatmap of occlusion sensitivity maps generated in Python.== %%POSTFIX%%The scale indicates the importan*
>%%LINK%%[[#^jqcuw3uuycg|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^jqcuw3uuycg


>%%
>```annotation-json
>{"created":"2022-09-07T09:23:29.207Z","updated":"2022-09-07T09:23:29.207Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":27765,"end":28087},{"type":"TextQuoteSelector","exact":"The orange arrows in (a, i) point to examples of cells withcondensed chromatin and perinuclear lightening zones the deep learning model picked up for prediction of mutated NPM1. The yellowarrows in (b, i) point to examples of cells with prominent nucleoli the deep learning model picked up for prediction of wild-type NPM1","prefix":"mportant for class predictions. ","suffix":".Fig. 5 Variant allele frequency"}]}]}
>```
>%%
>*%%PREFIX%%mportant for class predictions.%%HIGHLIGHT%% ==The orange arrows in (a, i) point to examples of cells withcondensed chromatin and perinuclear lightening zones the deep learning model picked up for prediction of mutated NPM1. The yellowarrows in (b, i) point to examples of cells with prominent nucleoli the deep learning model picked up for prediction of wild-type NPM1== %%POSTFIX%%.Fig. 5 Variant allele frequency*
>%%LINK%%[[#^8yg60huw8w8|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^8yg60huw8w8


>%%
>```annotation-json
>{"created":"2022-09-07T09:23:55.467Z","updated":"2022-09-07T09:23:55.467Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":25752,"end":26006},{"type":"TextQuoteSelector","exact":"Our study shows that DL can derive morphologic features fromcytomorphology that predict mutation status. Future work willfocus on other clinically important mutations and their morpho-logic imprint that DL may be able to pick up. In line with ourfindings","prefix":"dictive featuresby our DL model.","suffix":", a recent study showed that DL "}]}]}
>```
>%%
>*%%PREFIX%%dictive featuresby our DL model.%%HIGHLIGHT%% ==Our study shows that DL can derive morphologic features fromcytomorphology that predict mutation status. Future work willfocus on other clinically important mutations and their morpho-logic imprint that DL may be able to pick up. In line with ourfindings== %%POSTFIX%%, a recent study showed that DL*
>%%LINK%%[[#^irc60490ea|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^irc60490ea


>%%
>```annotation-json
>{"created":"2022-09-07T09:24:55.246Z","updated":"2022-09-07T09:24:55.246Z","document":{"title":"Deep learning detects acute myeloid leukemia and predicts NPM1 mutation status from bone marrow smears","link":[{"href":"urn:x-pdf:52e78e5cd6d18c937108db74c23703c1"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf"}],"documentFingerprint":"52e78e5cd6d18c937108db74c23703c1"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/41375_2021_Article_1408.pdf","selector":[{"type":"TextPositionSelector","start":29139,"end":30044},{"type":"TextQuoteSelector","exact":"Mori et al.[45] recently used CNN-guided detection of decreased granules—one of the most common dysplastic changes in MDS—and reporthigh accuracy for their classifier based on the ResNet-152.Accordingly, integrated analysis of more complex morphologiescan potentially be achieved by feature engineering using aknowledge bank of expert-annotated cells with sufficiently sizedtraining sets per morphologic feature (conceivably in the four- tofive-digit number range). Since many hematologic neoplasms arerare disease entities, the development of such a large databaserequires extensive cooperation and data sharing betweeninstitutions and countries to ideally provide an open-source bonemarrow database where independent ML models can be trainedon analogous to existing cancer data bases such as The CancerGenome Atlas [46]. Nevertheless, samples need to be properlyanonymized to warrant patient data safety","prefix":"es to the respective disorders. ","suffix":". If maintained andfunded proper"}]}]}
>```
>%%
>*%%PREFIX%%es to the respective disorders.%%HIGHLIGHT%% ==Mori et al.[45] recently used CNN-guided detection of decreased granules—one of the most common dysplastic changes in MDS—and reporthigh accuracy for their classifier based on the ResNet-152.Accordingly, integrated analysis of more complex morphologiescan potentially be achieved by feature engineering using aknowledge bank of expert-annotated cells with sufficiently sizedtraining sets per morphologic feature (conceivably in the four- tofive-digit number range). Since many hematologic neoplasms arerare disease entities, the development of such a large databaserequires extensive cooperation and data sharing betweeninstitutions and countries to ideally provide an open-source bonemarrow database where independent ML models can be trainedon analogous to existing cancer data bases such as The CancerGenome Atlas [46]. Nevertheless, samples need to be properlyanonymized to warrant patient data safety== %%POSTFIX%%. If maintained andfunded proper*
>%%LINK%%[[#^noung9eqvls|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^noung9eqvls
