---
annotation-target: pone.0269570 (1).pdf
---


>%%
>```annotation-json
>{"created":"2022-08-30T12:41:28.176Z","updated":"2022-08-30T12:41:28.176Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":694,"end":871},{"type":"TextQuoteSelector","exact":"However, there are concerns regarding theblack box problem: it is difficult to interpret why the prediction results are obtained usingdeep learning models from model parameters.","prefix":"on data and disease phenotypes. ","suffix":" New methods have been proposed "}]}]}
>```
>%%
>*%%PREFIX%%on data and disease phenotypes.%%HIGHLIGHT%% ==However, there are concerns regarding theblack box problem: it is difficult to interpret why the prediction results are obtained usingdeep learning models from model parameters.== %%POSTFIX%%New methods have been proposed*
>%%LINK%%[[#^8rfxyw7aubd|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^8rfxyw7aubd


>%%
>```annotation-json
>{"created":"2022-08-30T12:42:03.959Z","updated":"2022-08-30T12:42:03.959Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":1035,"end":1435},{"type":"TextQuoteSelector","exact":" SHapley Additive exPlanations (SHAP) to a deep learningmodel using graph convolutions of genetic pathways can provide pathway-level featureimportance for classification prediction of diffuse large B-cell lymphoma (DLBCL) geneexpression subtypes. Using Kyoto Encyclopedia of Genes and Genomes pathways, agraph convolutional network (GCN) model was implemented to construct graphs with nodesand edges.","prefix":"dy,we demonstrated that applying","suffix":" DLBCL datasets, including micro"}]}]}
>```
>%%
>*%%PREFIX%%dy,we demonstrated that applying%%HIGHLIGHT%% ==SHapley Additive exPlanations (SHAP) to a deep learningmodel using graph convolutions of genetic pathways can provide pathway-level featureimportance for classification prediction of diffuse large B-cell lymphoma (DLBCL) geneexpression subtypes. Using Kyoto Encyclopedia of Genes and Genomes pathways, agraph convolutional network (GCN) model was implemented to construct graphs with nodesand edges.== %%POSTFIX%%DLBCL datasets, including micro*
>%%LINK%%[[#^t06lrirmlh|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^t06lrirmlh


>%%
>```annotation-json
>{"created":"2022-08-30T12:42:29.471Z","updated":"2022-08-30T12:42:29.471Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":1692,"end":1836},{"type":"TextQuoteSelector","exact":"accuracy of 0.914, precision of 0.948, recall of 0.868, and F1 score of 0.906 inanalysis of the classification performance for the test datasets","prefix":"e model. The GCN modelshowed an ","suffix":". The pathways with high fea-tur"}]}]}
>```
>%%
>*%%PREFIX%%e model. The GCN modelshowed an%%HIGHLIGHT%% ==accuracy of 0.914, precision of 0.948, recall of 0.868, and F1 score of 0.906 inanalysis of the classification performance for the test datasets== %%POSTFIX%%. The pathways with high fea-tur*
>%%LINK%%[[#^ww1t6hjc4bc|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^ww1t6hjc4bc


>%%
>```annotation-json
>{"created":"2022-08-30T12:42:55.647Z","updated":"2022-08-30T12:42:55.647Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":2101,"end":2279},{"type":"TextQuoteSelector","exact":" predicting DLBCL sub-types. In conclusion, our GCN model for classifying DLBCL subtypes is useful forinterpreting important regulatory pathways that contribute to the prediction","prefix":"tance showed good performance in","suffix":".IntroductionGene expression pat"}]}]}
>```
>%%
>*%%PREFIX%%tance showed good performance in%%HIGHLIGHT%% ==predicting DLBCL sub-types. In conclusion, our GCN model for classifying DLBCL subtypes is useful forinterpreting important regulatory pathways that contribute to the prediction== %%POSTFIX%%.IntroductionGene expression pat*
>%%LINK%%[[#^oq5anr92s7|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^oq5anr92s7


>%%
>```annotation-json
>{"created":"2022-08-30T12:43:26.796Z","updated":"2022-08-30T12:43:26.796Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":3773,"end":4041},{"type":"TextQuoteSelector","exact":"Various statistical approaches including machine learning have been attempted to reveal theassociations between gene expressions and disease phenotypes. Deep learning is an emergingmachine learning method that enables highly accurate predictions of these associations.","prefix":"r preparation of the manuscript.","suffix":" How-ever, it is difficult to in"}]}]}
>```
>%%
>*%%PREFIX%%r preparation of the manuscript.%%HIGHLIGHT%% ==Various statistical approaches including machine learning have been attempted to reveal theassociations between gene expressions and disease phenotypes. Deep learning is an emergingmachine learning method that enables highly accurate predictions of these associations.== %%POSTFIX%%How-ever, it is difficult to in*
>%%LINK%%[[#^rd7iesszq28|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^rd7iesszq28


>%%
>```annotation-json
>{"created":"2022-08-30T12:43:49.430Z","updated":"2022-08-30T12:43:49.430Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":4203,"end":4293},{"type":"TextQuoteSelector","exact":". Explainability is crucial for pre-dictive models in medicine and other research fields. ","prefix":"box problem in deep learning [1]","suffix":"Several methods have been used t"}]}]}
>```
>%%
>*%%PREFIX%%box problem in deep learning [1]%%HIGHLIGHT%% ==. Explainability is crucial for pre-dictive models in medicine and other research fields.== %%POSTFIX%%Several methods have been used t*
>%%LINK%%[[#^n1oax5rcti|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^n1oax5rcti


>%%
>```annotation-json
>{"created":"2022-08-30T12:46:09.887Z","updated":"2022-08-30T12:46:09.887Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":5214,"end":5456},{"type":"TextQuoteSelector","exact":"Gene sets of genetic pathways, cytogenetic bands, andgene ontology have been manually curated from the databases. Genetic features such asexpression, mutation, and copy numbers were compared between two phenotypes to obtainenrichment scores, ","prefix":"o that of single gene analysis. ","suffix":"which were calculated from a cor"}]}]}
>```
>%%
>*%%PREFIX%%o that of single gene analysis.%%HIGHLIGHT%% ==Gene sets of genetic pathways, cytogenetic bands, andgene ontology have been manually curated from the databases. Genetic features such asexpression, mutation, and copy numbers were compared between two phenotypes to obtainenrichment scores,== %%POSTFIX%%which were calculated from a cor*
>%%LINK%%[[#^or0adigi05|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^or0adigi05


>%%
>```annotation-json
>{"created":"2022-08-30T12:47:12.180Z","updated":"2022-08-30T12:47:12.180Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":6053,"end":6516},{"type":"TextQuoteSelector","exact":" Some complex rela-tionships between features and phenotypes are ignored by GSEA, whereas deep learning canuse these relationships when making predictions. By interpreting the predictions of deeplearning, it may be possible to detect associations between gene expression patterns and phe-notypes that are not detected using GSEA. We used SHAP to interpret the deep learningmodel predictions because this method can summarize the feature importance across datasets","prefix":"ed toward a specific class [12].","suffix":".SHAP estimates the feature cont"}]}]}
>```
>%%
>*%%PREFIX%%ed toward a specific class [12].%%HIGHLIGHT%% ==Some complex rela-tionships between features and phenotypes are ignored by GSEA, whereas deep learning canuse these relationships when making predictions. By interpreting the predictions of deeplearning, it may be possible to detect associations between gene expression patterns and phe-notypes that are not detected using GSEA. We used SHAP to interpret the deep learningmodel predictions because this method can summarize the feature importance across datasets== %%POSTFIX%%.SHAP estimates the feature cont*
>%%LINK%%[[#^m2w4ff2jvi|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^m2w4ff2jvi


>%%
>```annotation-json
>{"created":"2022-08-30T12:51:11.859Z","text":"夏普利值方法，加入了顺序的概念，并以此来判断关键加入者，和他的边际贡献。","updated":"2022-08-30T12:51:11.859Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":6561,"end":6575},{"type":"TextQuoteSelector","exact":"Shapley value,","prefix":"es the feature contribution, or ","suffix":" to prediction on an additive sc"}]}]}
>```
>%%
>*%%PREFIX%%es the feature contribution, or%%HIGHLIGHT%% ==Shapley value,== %%POSTFIX%%to prediction on an additive sc*
>%%LINK%%[[#^w4up34vuaub|show annotation]]
>%%COMMENT%%
>夏普利值方法，加入了顺序的概念，并以此来判断关键加入者，和他的边际贡献。
>%%TAGS%%
>
^w4up34vuaub


>%%
>```annotation-json
>{"created":"2022-08-30T12:52:51.690Z","updated":"2022-08-30T12:52:51.690Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":8950,"end":8988},{"type":"TextQuoteSelector","exact":"and activated B-cell-like (ABC) type. ","prefix":"s on sharing data and materials.","suffix":"These subtypes are derived from "}]}]}
>```
>%%
>*%%PREFIX%%s on sharing data and materials.%%HIGHLIGHT%% ==and activated B-cell-like (ABC) type.== %%POSTFIX%%These subtypes are derived from*
>%%LINK%%[[#^j5115vna78|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^j5115vna78


>%%
>```annotation-json
>{"created":"2022-08-30T12:53:10.104Z","updated":"2022-08-30T12:53:10.104Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":9106,"end":9270},{"type":"TextQuoteSelector","exact":"The prognosis of patients withDLBCL treated with standard combination chemotherapy differs by subtype, and an optimaltreatment strategy is being developed [19, 20].","prefix":"erent gene expression patterns. ","suffix":" Recent studies revealed the mol"}]}]}
>```
>%%
>*%%PREFIX%%erent gene expression patterns.%%HIGHLIGHT%% ==The prognosis of patients withDLBCL treated with standard combination chemotherapy differs by subtype, and an optimaltreatment strategy is being developed [19, 20].== %%POSTFIX%%Recent studies revealed the mol*
>%%LINK%%[[#^15k64uknf5v|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^15k64uknf5v


>%%
>```annotation-json
>{"created":"2022-08-30T12:53:42.158Z","updated":"2022-08-30T12:53:42.158Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":9835,"end":10238},{"type":"TextQuoteSelector","exact":"ene expression data and clinical information in GSE31312 [25] and GSE10846 [19] wereobtained from the GEO database [18]. Each dataset included 498 and 414 patients withDLBCL, respectively. The same microarray platform (Affymetrix Human Genome U133 Plus2.0 Array, Santa Clara, CA, USA) was used for all gene expression data. In each study, thepatients with DLBCL subtypes [24] were labeled as GCB, ABC, o","prefix":" results of GSEA.MethodsDatasetG","suffix":"r unclassifiable in geneexpressi"}]}]}
>```
>%%
>*%%PREFIX%%results of GSEA.MethodsDatasetG%%HIGHLIGHT%% ==ene expression data and clinical information in GSE31312 [25] and GSE10846 [19] wereobtained from the GEO database [18]. Each dataset included 498 and 414 patients withDLBCL, respectively. The same microarray platform (Affymetrix Human Genome U133 Plus2.0 Array, Santa Clara, CA, USA) was used for all gene expression data. In each study, thepatients with DLBCL subtypes [24] were labeled as GCB, ABC, o== %%POSTFIX%%r unclassifiable in geneexpressi*
>%%LINK%%[[#^t4gmfexuf0q|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^t4gmfexuf0q


>%%
>```annotation-json
>{"created":"2022-08-30T12:54:05.971Z","updated":"2022-08-30T12:54:05.971Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":10674,"end":10767},{"type":"TextQuoteSelector","exact":"R 3.6.2 was used for normalization (The R Project for Statistical Computing,Vienna, Austria).","prefix":" eliminate noise foreach probe. ","suffix":" Next, the probe-level signal in"}]}]}
>```
>%%
>*%%PREFIX%%eliminate noise foreach probe.%%HIGHLIGHT%% ==R 3.6.2 was used for normalization (The R Project for Statistical Computing,Vienna, Austria).== %%POSTFIX%%Next, the probe-level signal in*
>%%LINK%%[[#^2r4uh2f5zb2|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^2r4uh2f5zb2


>%%
>```annotation-json
>{"created":"2022-08-30T12:55:23.231Z","updated":"2022-08-30T12:55:23.231Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":12499,"end":12703},{"type":"TextQuoteSelector","exact":"To create graphs representing genetic pathways, we first selected 186 pathways anno-tated in the KEGG [29] pathways (c2.cp.kegg.v7.3.symbols.gmt) in MSigDB [10, 11] and thegenes included in these pathways","prefix":" was constructed for thismodel. ","suffix":". Therefore, important pathways "}]}]}
>```
>%%
>*%%PREFIX%%was constructed for thismodel.%%HIGHLIGHT%% ==To create graphs representing genetic pathways, we first selected 186 pathways anno-tated in the KEGG [29] pathways (c2.cp.kegg.v7.3.symbols.gmt) in MSigDB [10, 11] and thegenes included in these pathways== %%POSTFIX%%. Therefore, important pathways*
>%%LINK%%[[#^dtzzaek9h8i|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^dtzzaek9h8i


>%%
>```annotation-json
>{"created":"2022-08-30T12:56:03.576Z","updated":"2022-08-30T12:56:03.576Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":13108,"end":13445},{"type":"TextQuoteSelector","exact":"he relationships between the genes in the KEGG pathways were deter-mined from the KEGG website and were set as edges between the nodes. Using these nodesand edges, we constructed 186 graphs G ¼ðV;EÞ, each representing a KEGG pathway. Thegraphs contained 12,797 nodes and 146,343 edges, including duplications among the differentpathways.","prefix":"EGG path-ways (S1A Fig). Next, t","suffix":" Graph convolutions were conduct"}]}]}
>```
>%%
>*%%PREFIX%%EGG path-ways (S1A Fig). Next, t%%HIGHLIGHT%% ==he relationships between the genes in the KEGG pathways were deter-mined from the KEGG website and were set as edges between the nodes. Using these nodesand edges, we constructed 186 graphs G ¼ðV;EÞ, each representing a KEGG pathway. Thegraphs contained 12,797 nodes and 146,343 edges, including duplications among the differentpathways.== %%POSTFIX%%Graph convolutions were conduct*
>%%LINK%%[[#^e4i5b85uyze|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^e4i5b85uyze


>%%
>```annotation-json
>{"created":"2022-08-30T12:58:01.773Z","updated":"2022-08-30T12:58:01.773Z","document":{"title":"Pathway importance by graph convolutional network and Shapley additive explanations in gene expression phenotype of diffuse large B-cell lymphoma","link":[{"href":"urn:x-pdf:00c133d820e5b02ea8cb17506ca510af"},{"href":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf"}],"documentFingerprint":"00c133d820e5b02ea8cb17506ca510af"},"uri":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","target":[{"source":"vault:/hematology-theroy/papers/hematology_deeplearning/pone.0269570 (1).pdf","selector":[{"type":"TextPositionSelector","start":22305,"end":22410},{"type":"TextQuoteSelector","exact":"The Pytorch 1.1, Deep Graph Library [32], and SHAP [9] as Python 3.8 packages were usedfor implementation","prefix":"utenormalized enrichment scores.","suffix":". All analyses were performed on"}]}]}
>```
>%%
>*%%PREFIX%%utenormalized enrichment scores.%%HIGHLIGHT%% ==The Pytorch 1.1, Deep Graph Library [32], and SHAP [9] as Python 3.8 packages were usedfor implementation== %%POSTFIX%%. All analyses were performed on*
>%%LINK%%[[#^wi3kqr98hrn|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^wi3kqr98hrn
