---
annotation-target: blood.2022015853.pdf
---


>%%
>```annotation-json
>{"created":"2022-09-22T14:18:17.210Z","updated":"2022-09-22T14:18:17.210Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":140,"end":157},{"type":"TextQuoteSelector","exact":"Genomic Profiling","prefix":"776-0545editorial@hematology.org","suffix":" for Clinical Decision Making in"}]}]}
>```
>%%
>*%%PREFIX%%776-0545editorial@hematology.org%%HIGHLIGHT%% ==Genomic Profiling== %%POSTFIX%%for Clinical Decision Making in*
>%%LINK%%[[#^qsh6v4x2vw|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^qsh6v4x2vw


>%%
>```annotation-json
>{"created":"2022-09-22T14:19:31.903Z","updated":"2022-09-22T14:19:31.903Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":18525,"end":18734},{"type":"TextQuoteSelector","exact":"increased our ability to detect somatic mutations through the use of targeted gene panels, whole exome sequencing (WES), whole genome sequencing (WGS), and whole transcriptome  sequencing  (WTS)  or  RNAseq.  ","prefix":"s in sequencing technology have ","suffix":"In  patients  with  myeloid  neo"}]}]}
>```
>%%
>*%%PREFIX%%s in sequencing technology have%%HIGHLIGHT%% ==increased our ability to detect somatic mutations through the use of targeted gene panels, whole exome sequencing (WES), whole genome sequencing (WGS), and whole transcriptome  sequencing  (WTS)  or  RNAseq.== %%POSTFIX%%In  patients  with  myeloid  neo*
>%%LINK%%[[#^wqg9ib576f|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^wqg9ib576f


>%%
>```annotation-json
>{"created":"2022-09-22T14:20:07.884Z","updated":"2022-09-22T14:20:07.884Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":20320,"end":20498},{"type":"TextQuoteSelector","exact":" This  manuscript  is  meant  to  be  a  practical  guide  for  the application  of  genomic  methods  in  the  clinical  evaluation  of  myeloid  neoplasms  and  acute leukemia.","prefix":"bed  below  and  in  Table 1.   ","suffix":"  Conventional Methods Chromosom"}]}]}
>```
>%%
>*%%PREFIX%%bed  below  and  in  Table 1.%%HIGHLIGHT%% ==This  manuscript  is  meant  to  be  a  practical  guide  for  the application  of  genomic  methods  in  the  clinical  evaluation  of  myeloid  neoplasms  and  acute leukemia.== %%POSTFIX%%Conventional Methods Chromosom*
>%%LINK%%[[#^1fqnpav5olh|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^1fqnpav5olh


>%%
>```annotation-json
>{"created":"2022-09-22T14:20:22.768Z","updated":"2022-09-22T14:20:22.768Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":20521,"end":20550},{"type":"TextQuoteSelector","exact":"Chromosome  banding  analysis","prefix":"leukemia.  Conventional Methods ","suffix":":  Karyotyping  remains  the  mo"}]}]}
>```
>%%
>*%%PREFIX%%leukemia.  Conventional Methods%%HIGHLIGHT%% ==Chromosome  banding  analysis== %%POSTFIX%%:  Karyotyping  remains  the  mo*
>%%LINK%%[[#^c76oie297a6|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^c76oie297a6


>%%
>```annotation-json
>{"created":"2022-09-22T14:22:42.980Z","updated":"2022-09-22T14:22:42.980Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":20802,"end":21123},{"type":"TextQuoteSelector","exact":"The  main limitations are the requirement for live culturable cells, low resolution (5-10 Mbases), and low sensitivity  (abnormalities  present  in  5-10%  of  cells  or  an  analytical  sensitivity  of  ~10-1).  Turnaround  times  are  generally  between  2-21  days  and  may  vary  considerably  between laboratories. ","prefix":"inversions)  abnormalities1-3.  ","suffix":" Fluorescence  in  situ  hybridi"}]}]}
>```
>%%
>*%%PREFIX%%inversions)  abnormalities1-3.%%HIGHLIGHT%% ==The  main limitations are the requirement for live culturable cells, low resolution (5-10 Mbases), and low sensitivity  (abnormalities  present  in  5-10%  of  cells  or  an  analytical  sensitivity  of  ~10-1).  Turnaround  times  are  generally  between  2-21  days  and  may  vary  considerably  between laboratories.== %%POSTFIX%%Fluorescence  in  situ  hybridi*
>%%LINK%%[[#^gcs62l45u56|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^gcs62l45u56


>%%
>```annotation-json
>{"created":"2022-09-22T14:22:50.692Z","updated":"2022-09-22T14:22:50.692Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":21124,"end":21170},{"type":"TextQuoteSelector","exact":"Fluorescence  in  situ  hybridization  (FISH):","prefix":"derably  between laboratories.  ","suffix":"  FISH  is  often  used  to  com"}]}]}
>```
>%%
>*%%PREFIX%%derably  between laboratories.%%HIGHLIGHT%% ==Fluorescence  in  situ  hybridization  (FISH):== %%POSTFIX%%FISH  is  often  used  to  com*
>%%LINK%%[[#^migzlb0jt0g|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^migzlb0jt0g


>%%
>```annotation-json
>{"created":"2022-09-22T14:23:03.076Z","updated":"2022-09-22T14:23:03.076Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":21353,"end":21646},{"type":"TextQuoteSelector","exact":"FISH  probes  can  only  identify  genomic  events  at  specific targeted regions, but is more sensitive than cytogenetics (abnormalities in 1-5% of cells or an analytical sensitivity of ~10-2) and can detect cytogenetically cryptic abnormalities.4,5 Turnaround times are generally 1-3 days.  ","prefix":"dividing  cells  (interphase).  ","suffix":"Downloaded from http://ashpublic"}]}]}
>```
>%%
>*%%PREFIX%%dividing  cells  (interphase).%%HIGHLIGHT%% ==FISH  probes  can  only  identify  genomic  events  at  specific targeted regions, but is more sensitive than cytogenetics (abnormalities in 1-5% of cells or an analytical sensitivity of ~10-2) and can detect cytogenetically cryptic abnormalities.4,5 Turnaround times are generally 1-3 days.== %%POSTFIX%%Downloaded from http://ashpublic*
>%%LINK%%[[#^srygb29yrbm|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^srygb29yrbm


>%%
>```annotation-json
>{"created":"2022-09-22T14:23:23.419Z","updated":"2022-09-22T14:23:23.419Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":21797,"end":21831},{"type":"TextQuoteSelector","exact":"Chromosomal  microarrays  (CMA):  ","prefix":"by guest on 22 September 2022  8","suffix":"CMAs  are  typically  used  to  "}]}]}
>```
>%%
>*%%PREFIX%%by guest on 22 September 2022  8%%HIGHLIGHT%% ==Chromosomal  microarrays  (CMA):== %%POSTFIX%%CMAs  are  typically  used  to*
>%%LINK%%[[#^mzle2w1lhc|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^mzle2w1lhc


>%%
>```annotation-json
>{"created":"2022-09-22T14:23:41.594Z","updated":"2022-09-22T14:23:41.594Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":22909,"end":22938},{"type":"TextQuoteSelector","exact":"Optical Genome Mapping (OGM):","prefix":"to determine DNA copy state.7   ","suffix":" Optical genome mapping methods "}]}]}
>```
>%%
>*%%PREFIX%%to determine DNA copy state.7%%HIGHLIGHT%% ==Optical Genome Mapping (OGM):== %%POSTFIX%%Optical genome mapping methods*
>%%LINK%%[[#^vbzmybdyyxh|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^vbzmybdyyxh


>%%
>```annotation-json
>{"created":"2022-09-22T14:23:54.358Z","updated":"2022-09-22T14:23:54.358Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":23097,"end":23205},{"type":"TextQuoteSelector","exact":"genomic DNA to identify structural variants such as translocations, inversions, and copy number alterations.","prefix":"ts  of  high  molecular  weight ","suffix":"8  While not widely used in the "}]}]}
>```
>%%
>*%%PREFIX%%ts  of  high  molecular  weight%%HIGHLIGHT%% ==genomic DNA to identify structural variants such as translocations, inversions, and copy number alterations.== %%POSTFIX%%8  While not widely used in the*
>%%LINK%%[[#^4hpdn8oda72|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^4hpdn8oda72


>%%
>```annotation-json
>{"created":"2022-09-22T14:24:22.557Z","updated":"2022-09-22T14:24:22.557Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":24448,"end":24733},{"type":"TextQuoteSelector","exact":"qPCR and dPCR methods are generally suited to detect specific recurrent genetic alterations such  as  single  gene  mutations  (e.g.  JAK2  p.V617F,  KIT  p.D816V)  and  distinct  fusions  (e.g. BCR::ABL1) for diagnosis and disease monitoring.  Turnaround times are generally 2-5 days.","prefix":"lls or a sensitivity of 10-4).  ","suffix":"   Sanger  sequencing:  Sanger  "}]}]}
>```
>%%
>*%%PREFIX%%lls or a sensitivity of 10-4).%%HIGHLIGHT%% ==qPCR and dPCR methods are generally suited to detect specific recurrent genetic alterations such  as  single  gene  mutations  (e.g.  JAK2  p.V617F,  KIT  p.D816V)  and  distinct  fusions  (e.g. BCR::ABL1) for diagnosis and disease monitoring.  Turnaround times are generally 2-5 days.== %%POSTFIX%%Sanger  sequencing:  Sanger*
>%%LINK%%[[#^r6ofyobag9o|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^r6ofyobag9o


>%%
>```annotation-json
>{"created":"2022-09-22T14:24:44.058Z","updated":"2022-09-22T14:24:44.058Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":24736,"end":25257},{"type":"TextQuoteSelector","exact":"Sanger  sequencing:  Sanger  sequencing  detects  small  gene-level  DNA  variation  from  PCR-amplified DNA fragments (<1 kb). Individual DNA bases are detected by electrophoresis due to the random incorporation of fluorescently labeled chain-terminating dideoxynucleotides by DNA polymerase during in vitro DNA replication. Sanger sequencing is generally used to detect gene mutations  confined  to  single  exons  (e.g.  CEBPA,  CALR)  and  has  a  relatively  low  analytic sensitivity  of  approximately  20%  (>10-1","prefix":"times are generally 2-5 days.   ","suffix":"),  but  can  be  turned  around"}]}]}
>```
>%%
>*%%PREFIX%%times are generally 2-5 days.%%HIGHLIGHT%% ==Sanger  sequencing:  Sanger  sequencing  detects  small  gene-level  DNA  variation  from  PCR-amplified DNA fragments (<1 kb). Individual DNA bases are detected by electrophoresis due to the random incorporation of fluorescently labeled chain-terminating dideoxynucleotides by DNA polymerase during in vitro DNA replication. Sanger sequencing is generally used to detect gene mutations  confined  to  single  exons  (e.g.  CEBPA,  CALR)  and  has  a  relatively  low  analytic sensitivity  of  approximately  20%  (>10-1== %%POSTFIX%%),  but  can  be  turned  around*
>%%LINK%%[[#^l3808hlytc|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^l3808hlytc


>%%
>```annotation-json
>{"created":"2022-09-22T14:24:53.010Z","updated":"2022-09-22T14:24:53.010Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":25362,"end":25409},{"type":"TextQuoteSelector","exact":"Next Generation Sequencing (NGS)-based methods ","prefix":"quencing (NGS) based methods.   ","suffix":"General  Concepts:  Building  up"}]}]}
>```
>%%
>*%%PREFIX%%quencing (NGS) based methods.%%HIGHLIGHT%% ==Next Generation Sequencing (NGS)-based methods== %%POSTFIX%%General  Concepts:  Building  up*
>%%LINK%%[[#^ccmujd28s4a|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^ccmujd28s4a


>%%
>```annotation-json
>{"created":"2022-09-22T14:25:22.065Z","updated":"2022-09-22T14:25:22.065Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":26646,"end":27243},{"type":"TextQuoteSelector","exact":"The  abundance  of  a  detected  variant  is  generally  represented  as  a  variant  allele  frequency (VAF, Figure 2A), which represents the ratio of sequencing reads that contain a variant at a given position in the genome divided by the total number of reads at the position. VAFs are considered a ‘semi-quantitative’ measure, as the exact methods used to calculate VAFs, VAF precision, and VAF accuracy will differ slightly between laboratories. Additionally, chromosomal aneuploidy, LOH, or gene amplification/deletion can skew both inherited and somatic variant VAFs either higher or lower.","prefix":"gnosis as well as monitoring.   ","suffix":" The methods used to separate so"}]}]}
>```
>%%
>*%%PREFIX%%gnosis as well as monitoring.%%HIGHLIGHT%% ==The  abundance  of  a  detected  variant  is  generally  represented  as  a  variant  allele  frequency (VAF, Figure 2A), which represents the ratio of sequencing reads that contain a variant at a given position in the genome divided by the total number of reads at the position. VAFs are considered a ‘semi-quantitative’ measure, as the exact methods used to calculate VAFs, VAF precision, and VAF accuracy will differ slightly between laboratories. Additionally, chromosomal aneuploidy, LOH, or gene amplification/deletion can skew both inherited and somatic variant VAFs either higher or lower.== %%POSTFIX%%The methods used to separate so*
>%%LINK%%[[#^gfxfzcxj93m|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^gfxfzcxj93m


>%%
>```annotation-json
>{"created":"2022-09-22T14:26:15.549Z","updated":"2022-09-22T14:26:15.549Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":28888,"end":29166},{"type":"TextQuoteSelector","exact":"n addition to inter-laboratory differences in VAFs and assignment of somatic status to variants, variant  classification  and  annotation,  including  assigning variants  to different  ‘tiers’  based  on pathogenicity or clinical significance, may differ between laboratories.  ","prefix":" guest on 22 September 2022  11I","suffix":"In general, it is advised that  "}]}]}
>```
>%%
>*%%PREFIX%%guest on 22 September 2022  11I%%HIGHLIGHT%% ==n addition to inter-laboratory differences in VAFs and assignment of somatic status to variants, variant  classification  and  annotation,  including  assigning variants  to different  ‘tiers’  based  on pathogenicity or clinical significance, may differ between laboratories.== %%POSTFIX%%In general, it is advised that*
>%%LINK%%[[#^o7apwumifs8|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^o7apwumifs8


>%%
>```annotation-json
>{"created":"2022-09-22T14:26:22.186Z","updated":"2022-09-22T14:26:22.186Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":29791,"end":29814},{"type":"TextQuoteSelector","exact":"Targeted  Gene  Panels:","prefix":", and relevant clinical data.   ","suffix":" Driver mutations (recurrent som"}]}]}
>```
>%%
>*%%PREFIX%%, and relevant clinical data.%%HIGHLIGHT%% ==Targeted  Gene  Panels:== %%POSTFIX%%Driver mutations (recurrent som*
>%%LINK%%[[#^2iq7z6nxa1z|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^2iq7z6nxa1z


>%%
>```annotation-json
>{"created":"2022-09-22T14:29:23.673Z","updated":"2022-09-22T14:29:23.673Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":30079,"end":30800},{"type":"TextQuoteSelector","exact":"Targeted  panels,  used  most  frequently  in  clinical  laboratories,  direct  sequencing  to  specific genes  or  genetic  regions  that  have  defined  clinical  relevance  and  dictate  clinical management.20,21  Key  genes  that  should  be  included  in  sequencing  panels  for  different diagnostic entities are summarized in Table 2. Enrichment strategies (Figure 2C) include the capture of genetic regions of interest though hybridization (hybrid capture using DNA or RNA probes) or PCR amplification (amplicon enrichment). This provides critical benefits over broad exome  (WES)  and  genome  (WGS)  sequencing  by  increasing  sensitivity  in  clinically  relevant regions and by decreasing sequencing cost.  ","prefix":"  by  small  gene  panels.18,19 ","suffix":"Depending on the design, targete"}]}]}
>```
>%%
>*%%PREFIX%%by  small  gene  panels.18,19%%HIGHLIGHT%% ==Targeted  panels,  used  most  frequently  in  clinical  laboratories,  direct  sequencing  to  specific genes  or  genetic  regions  that  have  defined  clinical  relevance  and  dictate  clinical management.20,21  Key  genes  that  should  be  included  in  sequencing  panels  for  different diagnostic entities are summarized in Table 2. Enrichment strategies (Figure 2C) include the capture of genetic regions of interest though hybridization (hybrid capture using DNA or RNA probes) or PCR amplification (amplicon enrichment). This provides critical benefits over broad exome  (WES)  and  genome  (WGS)  sequencing  by  increasing  sensitivity  in  clinically  relevant regions and by decreasing sequencing cost.== %%POSTFIX%%Depending on the design, targete*
>%%LINK%%[[#^qvlt6g68u4|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^qvlt6g68u4


>%%
>```annotation-json
>{"created":"2022-09-22T14:29:42.789Z","updated":"2022-09-22T14:29:42.789Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":31661,"end":31981},{"type":"TextQuoteSelector","exact":"Genomic  sequencing  (WES  and  WGS)  and  transcriptomic  sequencing  (WTS  or  RNASeq): Broad genomic sequencing assays allow for the detection of genomic alterations anywhere in the coding genome (WES whole exome sequencing, interrogating 1-2% of the whole genome) or  entire  genome  (WGS-whole  genome  sequencing).","prefix":"nels are generally 5-14 days.   ","suffix":"24,25  WGS  and  WES  entail  hi"}]}]}
>```
>%%
>*%%PREFIX%%nels are generally 5-14 days.%%HIGHLIGHT%% ==Genomic  sequencing  (WES  and  WGS)  and  transcriptomic  sequencing  (WTS  or  RNASeq): Broad genomic sequencing assays allow for the detection of genomic alterations anywhere in the coding genome (WES whole exome sequencing, interrogating 1-2% of the whole genome) or  entire  genome  (WGS-whole  genome  sequencing).== %%POSTFIX%%24,25  WGS  and  WES  entail  hi*
>%%LINK%%[[#^j5o9x0tlj2h|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^j5o9x0tlj2h


>%%
>```annotation-json
>{"created":"2022-09-22T14:29:57.019Z","updated":"2022-09-22T14:29:57.019Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":33113,"end":33165},{"type":"TextQuoteSelector","exact":"Molecular measurable residual disease (MRD) methods ","prefix":"into routine clinical testing.  ","suffix":" qPCR and FISH: The oldest and m"}]}]}
>```
>%%
>*%%PREFIX%%into routine clinical testing.%%HIGHLIGHT%% ==Molecular measurable residual disease (MRD) methods== %%POSTFIX%%qPCR and FISH: The oldest and m*
>%%LINK%%[[#^tqrwrhhbchq|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^tqrwrhhbchq


>%%
>```annotation-json
>{"created":"2022-09-22T14:30:09.007Z","updated":"2022-09-22T14:30:09.007Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":33166,"end":33458},{"type":"TextQuoteSelector","exact":"qPCR and FISH: The oldest and most established method for monitoring MRD in hematopoietic neoplasms  rely  on  the  detection  of  previously  identified  translocations  (PML::RARA, BCR::ABL1, RUNX1::RUNX1T1)  or  recurrent  insertions/deletion  (NPM1  exon  11  mutation (ENST00000296930); ","prefix":"residual disease (MRD) methods  ","suffix":"the same mutation may also be an"}]}]}
>```
>%%
>*%%PREFIX%%residual disease (MRD) methods%%HIGHLIGHT%% ==qPCR and FISH: The oldest and most established method for monitoring MRD in hematopoietic neoplasms  rely  on  the  detection  of  previously  identified  translocations  (PML::RARA, BCR::ABL1, RUNX1::RUNX1T1)  or  recurrent  insertions/deletion  (NPM1  exon  11  mutation (ENST00000296930);== %%POSTFIX%%the same mutation may also be an*
>%%LINK%%[[#^253tz38x7ku|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^253tz38x7ku


>%%
>```annotation-json
>{"created":"2022-09-22T14:30:43.608Z","updated":"2022-09-22T14:30:43.608Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":33858,"end":34009},{"type":"TextQuoteSelector","exact":"In AML high sensitivity monitoring for highly recurrent NPM1 exon 11 gene mutation can be accomplished by qPCR with a sensitivity of ~10-3 or lower.30 ","prefix":"10-2 and 10-6, respectively.29  ","suffix":" High  sensitivity  sequencing  "}]}]}
>```
>%%
>*%%PREFIX%%10-2 and 10-6, respectively.29%%HIGHLIGHT%% ==In AML high sensitivity monitoring for highly recurrent NPM1 exon 11 gene mutation can be accomplished by qPCR with a sensitivity of ~10-3 or lower.30== %%POSTFIX%%High  sensitivity  sequencing*
>%%LINK%%[[#^byiplz9c4n|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^byiplz9c4n


>%%
>```annotation-json
>{"created":"2022-09-22T14:31:16.750Z","updated":"2022-09-22T14:31:16.750Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":34010,"end":34064},{"type":"TextQuoteSelector","exact":"High  sensitivity  sequencing  for  somatic  variants:","prefix":"nsitivity of ~10-3 or lower.30  ","suffix":" The sensitivity and specificity"}]}]}
>```
>%%
>*%%PREFIX%%nsitivity of ~10-3 or lower.30%%HIGHLIGHT%% ==High  sensitivity  sequencing  for  somatic  variants:== %%POSTFIX%%The sensitivity and specificity*
>%%LINK%%[[#^2mtl9drvi5w|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^2mtl9drvi5w


>%%
>```annotation-json
>{"created":"2022-09-22T14:31:25.760Z","updated":"2022-09-22T14:31:25.760Z","document":{"title":"blood.2022015853.pdf","link":[{"href":"urn:x-pdf:ba26ba5c3fef66ad63508c622a8605f9"},{"href":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf"}],"documentFingerprint":"ba26ba5c3fef66ad63508c622a8605f9"},"uri":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","target":[{"source":"vault:/hematology-theroy/papers/20220922/blood.2022015853.pdf","selector":[{"type":"TextPositionSelector","start":34792,"end":34869},{"type":"TextQuoteSelector","exact":"T-cell  receptor  (TR)  and  Immunoglobulin  (IG)  NGS-based  MRD  approaches","prefix":"in a patient-specific manner.   ","suffix":":  NGS  of  the hypervariable re"}]}]}
>```
>%%
>*%%PREFIX%%in a patient-specific manner.%%HIGHLIGHT%% ==T-cell  receptor  (TR)  and  Immunoglobulin  (IG)  NGS-based  MRD  approaches== %%POSTFIX%%:  NGS  of  the hypervariable re*
>%%LINK%%[[#^l3iid7ziv4e|show annotation]]
>%%COMMENT%%
>
>%%TAGS%%
>
^l3iid7ziv4e
